src/how-to/create-a-github-repository-secret.html

```diff
@@ -0,0 +1,159 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="How to create a repository secret for a personal access token in GitHub"
+    />
+    <link rel="stylesheet" href="/src/styles/common.css" />
+    <title>How to create a repository secret in GitHub</title>
+  </head>
+  <body class="body">
+    <header>
+      <h1>Create a repository secret in GitHub</h1>
+    </header>
+    <main class="main">
+      <article>
+        <p><em>Task</em>: create a GitHub repository secret.</p>
+        <p>
+          <em>Note</em>: The example creates a repository secret
+          <code>ASSIGN_REVIEWERS</code> for creating the GitHub action
+          <a href="/src/how-to/create-a-github-action.html">Auto Assign</a>.
+        </p>
+        <section aria-labelledby="create-pat">
+          <h2 id="create-pat">Create a Personal Access Token (PAT)</h2>
+          <p>
+            In order to create a repository secret, it's first necessary to
+            create a <abbr title="Personal Access Token">PAT</abbr>:
+          </p>
+          <ol>
+            <li>
+              Go to your GitHub
+              <a href="https://github.com/settings/apps">developer settings</a>
+            </li>
+            <li>
+              Navigate to Developer Settings > Personal Access Tokens > Tokens
+              (classic)
+              <div>
+                <a href="https://github.com/settings/tokens">tokens</a>
+              </div>
+            </li>
+            <li>Click "Generate new token (classic)"</li>
+            <li>Give it a descriptive e.g. "Auto Assign Reviewers"</li>
+            <li>
+              Select the <strong>workflow</strong> scope for the necessary
+              permissions (this will automatically select
+              <strong>repo</strong> which is also required)
+            </li>
+            <li>Copy the token immediately.</li>
+          </ol>
+        </section>
+        <section aria-labelledby="create-repo-secret">
+          <h2 id="create-repo-secret">How to create a repository secret</h2>
+          <p>
+            Once you have copied the
+            <abbr title="Personal Access Token">PAT</abbr>:
+          </p>
+          <ol>
+            <li>
+              Navigate to Settings > Secrets and variables > Actions
```

Comment:
You need to specify that this is at the repo settings, not the profile settings like for the PAT, it took me a while to find the secrets.

Reply:
Agreed. The process is convoluted but makes sense once you've done it. I've added a link here too to the Appendix because Claude did a good job of summarising PATs and secrets.